CONTRIBUTING.md

```diff
@@ -7,44 +7,103 @@ We welcomes all contributions! The only requirement we have is that, like many o
 [Please submit your CLA electronically using our Google form](https://docs.google.com/forms/d/e/1FAIpQLSeQ9ROFaTu9pyrmPhXc-dEnLD84DbLuT_-tPNZDOL9J10tOKQ/viewform) so we can accept your submissions. The GitHub username you file there will need to match that of your Pull Requests. If you have any questions or cannot file the CLA electronically, you can email help@realm.io.
 
 
-# How to build locally:
-
+## How to build locally
 
 ### Prerequisites
 
 - Swig. On Mac this can be installed using Homebrew: `brew install swig`.
 - CMake 3.18.1 or above. Can be installed through the Android SDK Manager.
 - Java 11.
 
-
-### Commands to build from source
+### Obtaining the source code 
 
 Checkout repo:
 ```
 git clone --recursive  https://github.com/realm/realm-kotlin.git 
 ```
 
-Build library:
+### Building and running test
+
+To support simultaneous development of the SDK and test, while still selectively allowing to run
```

Comment:
This is also described under "Writing Tests", maybe just keep it there?

Reply:
Yeah. I tried to put the motivation into this section ... but has not reworked it in to the _"Advanced project setup"_ section instead.